ako-infra/ingestion/avi_infra_controller.go

```diff
@@ -195,13 +196,32 @@ func ConfigureSeGroup(client *clients.AviClient, seGroup *models.ServiceEngineGr
 
 }
 
+// ConfigureSeGroup creates the SE group with the supplied properties, alters just the SE group name and the labels.
+func CheckSeGroup(client *clients.AviClient) bool {
+	// Change the name of the SE group
+	segroupName := lib.GetClusterID()
+
+	uri := "/api/serviceenginegroup/?name=" + segroupName
+	response := models.ServiceEngineGroupAPIResponse{}
+	err := lib.AviGet(client, uri, &response)
+	if err != nil {
+		utils.AviLog.Warnf("Error during Get call for the SE group :%v", err.Error())
+		return false
+	}
+	utils.AviLog.Infof("Found Service Engine Group :%v", segroupName)
+	return true
+}
+
 func (a *AviControllerInfra) AnnotateSystemNamespace(seGroup string, cloudName string) bool {
-	nsName := utils.VMWARE_SYSTEM_AKO
+	nsName := utils.GetAKONamespace()
 	nsObj, err := a.cs.CoreV1().Namespaces().Get(context.TODO(), nsName, metav1.GetOptions{})
```

Comment:
can we use the informer cache call here as well?

Reply:
We don't have access to informers here, and it should not be updated frequently, so I think its fine to use client here.